doc/Language/operators.pod6

```diff
@@ -2131,7 +2099,7 @@ X<Numeric greater than operator>.
 Coerces both arguments to L<Real|/type/Real> (if necessary); returns C<True> if the first argument
 is larger than the second.
 
-=head2 infix C«>=»
+=head2 infix C«>=», infix C«≥»
```

Comment:
OK, I see what you're doing, but if you add this here, you'll need to add the definition(s) below. Or maybe eliminate them...